phalcon/session/adapter.zep

```diff
@@ -260,9 +260,9 @@ abstract class Adapter implements AdapterInterface
 	 * );
 	 *</code>
 	 */
-	public function destroy(boolean removeData = false) -> boolean
+	public function destroy(var removeData = null) -> boolean
```

Comment:
Please see https://github.com/phalcon/cphalcon/issues/12833

Reply:
yea this is bad I know. But as you know it was "strange". This is what I came up as an idea to keep it working and not break anything and in 4.0.x it can be dramatically changed